vNext/shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -3,6 +3,9 @@
 "use strict";
 
 export class CoreUtils {
+    private static _canUseCookies: boolean;
+    private static document: any = typeof document !== "undefined" ? document : {};
```

Comment:
move back to common. Core should not take dependency on document object. Keep CoreUtils solely for what is user specified for cookie usage (isCookieUsageDisabledByUser, something like that).

Reply:
moved stuff that uses document back to common. Only `disableCookies` is in CoreUtils now.